opennms-dao-mock/src/main/java/org/opennms/netmgt/dao/mock/MockOutageDao.java

```diff
@@ -110,4 +111,9 @@ public int countOutagesByNode() {
     public List<HeatMapElement> getHeatMapItemsForEntity(String entityNameColumn, String entityIdColumn, String restrictionColumn, String restrictionValue, String... groupByColumns) {
         throw new UnsupportedOperationException("Not yet implemented!");
     }
+
+    @Override
+    public Collection<OnmsOutage> getStatusChangesForApplicationIdBetween(Date startDate, Date endDate, Integer applicationId) {
```

Comment:
Not sure if the method name makes it clear that we are talking about perspective!=NULL ?

Reply:
Hmmm... I hope it's clear since the term Application is part of the method-name.